src/index.js

```diff
@@ -6,17 +6,16 @@ import { browserHistory } from 'react-router';
 import Routes from './routes';
 import thunk from 'redux-thunk';
 import createLogger from 'redux-logger';
+
 import rootReducer from './reducers/rootReducer';
```

Comment:
This is going unused at this point, so it makes sense to remove it before this gets merged.